tests/utils.py

```diff
@@ -1,5 +1,6 @@
 import os
 import sys
+import json
```

Comment:
This seems unused

Reply:
line 11:
`    stream = json.loads(line.decode('utf-8')).get('stream')`